docs/excel/make-custom-functions-compatible-with-xll-udf.md

```diff
@@ -1,30 +1,27 @@
 ---
 title: Extend custom functions with XLL user-defined functions
 description: 'Enable compatibility with Excel XLL user-defined functions that have equivalent functionality to your custom functions'
-ms.date: 03/09/2021
+ms.date: 08/23/2021
 localization_priority: Normal
 ---
 
 # Extend custom functions with XLL user-defined functions
 
-If you have existing Excel XLLs, you can build equivalent custom functions in an Excel add-in to extend your solution features to other platforms such as online or on a Mac. However, Excel add-ins don't have all of the functionality available in XLLs. Depending on the functionality your solution uses, the XLL may provide a better experience than the Excel add-in custom functions in Excel on Windows.
+If you have existing Excel XLL add-ins, you can build equivalent custom function add-ins using the Excel JavaScript API to extend your solution features to other platforms, such as Excel on the web or on a Mac. However, Excel JavaScript API add-ins don't have all of the functionality available in XLL add-ins. Depending on the functionality your solution uses, the XLL add-in may provide a better experience than the Excel JavaScript API add-in in Excel on Windows.
 
 > [!NOTE]
-> COM add-in and XLL UDF compatibility is supported by the following platforms, when connected to a Microsoft 365 subscription.
->
-> - Excel on the web
-> - Excel on Windows (version 1904 or later)
-> - Excel on Mac (version 13.329 or later)
->
-> To use COM add-in and XLL UDF compatibility within Excel on the web, login by using either your Microsoft 365 subscription or a [Microsoft account](https://account.microsoft.com/account). If you don't already have a Microsoft 365 subscription, you can a free, 90-day renewable Microsoft 365 subscription by joining the [Microsoft 365 developer program](https://developer.microsoft.com/office/dev-program).
+> An XLL add-in is an Excel add-in file with the file extension **.xll**. An XLL file is a type of dynamic link library (DLL) file that can only be opened by Excel. XLL add-in files must be written in C or C++. 
```

Comment:
Can we get this definition earlier? Maybe it could be a note preceding the top paragraph, so readers know what this is before we start talking about it. A link to dev docs for XLL would be nice too.